index.html

```diff
@@ -48,5 +48,164 @@ <h2 class="title">I'm a Software Developer</h2>
             <img class="imgh" src="img/V13.png">
         </div>
     </section>
+    <section class="work">
+        <div class="work-title1">
+            <h1 class="work-head">My Recent Works</h1>
+            <img class="work1-img" src="img/Img1.png">
+            <h2 class="work-head2">Multi-Post Stories</h2>
+            <p class="work1-para">
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+            <div class="work-menu">
+                <ul class="work1-list">
+                    <li class="p">css</li>
+                    <li>html</li>
+                    <li>bookstrap</li>
+                    <li>Ruby</li>
+                </ul>
+            </div>
+            <a href="#" class="one btn">See Project</a>
+        </div>
+        <div class="work-title2">
+            <h1 class="work-head22">Professional Art Printing Data</h1>
+            <p class="work2-para">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
```

Comment:
[ ] kindly add a background image for your cards to match the design provided.🤓
The Figma design : 👇
![design ta3hm](https://user-images.githubusercontent.com/62156607/198575092-9698a581-5ca1-4edd-8d97-2c6f4f3c8481.PNG)

Your design :👇
![designta3ou](https://user-images.githubusercontent.com/62156607/198575103-864ff7f7-db56-4fe9-b8bf-4739bc61ff97.PNG)

Reply:
> [ ] kindly add a background image for your cards to match the design provided.🤓 The Figma design : 👇 ![design ta3hm](https://user-images.githubusercontent.com/62156607/198575092-9698a581-5ca1-4edd-8d97-2c6f4f3c8481.PNG)
> 
> Your design :👇 ![designta3ou](https://user-images.githubusercontent.com/62156607/198575103-864ff7f7-db56-4fe9-b8bf-4739bc61ff97.PNG)





I did put the background image, but when i run for error in the linters, i was asked to put a string in the URL.

![style](https://user-images.githubusercontent.com/77361109/198586925-25de5894-6c76-42ef-a320-c5854e72539d.PNG)

Reply:
> > [ ] kindly add a background image for your cards to match the design provided.🤓 The Figma design : 👇 ![design ta3hm](https://user-images.githubusercontent.com/62156607/198575092-9698a581-5ca1-4edd-8d97-2c6f4f3c8481.PNG)
> > Your design :👇 ![designta3ou](https://user-images.githubusercontent.com/62156607/198575103-864ff7f7-db56-4fe9-b8bf-4739bc61ff97.PNG)
> 
> I did put the background image, but when i run for error in the linters, i was asked to put a string in the URL.
> 
> ![style](https://user-images.githubusercontent.com/77361109/198586925-25de5894-6c76-42ef-a320-c5854e72539d.PNG)

Okay, I have fixed it